File: app/src/main/java/io/branch/demoapp/CustomApplicationClass.java
package io.branch.demoapp;

import android.app.Application;

import io.branch.referral.Branch;

/**
 * Created by Evan Groth on 11/8/16.
 */

public class CustomApplicationClass extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        Branch.getAutoInstance(this);
    }
}

File: app/src/main/java/io/branch/demoapp/CustomApplicationClass.kt
package io.branch.demoapp

import android.app.Application

import io.branch.referral.Branch

/**
 * Created by Evan Groth on 11/8/16.
 */

class CustomApplicationClass : Application() {
    override fun onCreate() {
        super.onCreate()
        Branch.enableLogging()
        Branch.getAutoInstance(this)
    }
}
